src/pyOpenMS/pxds/IsotopeLabelingMDVs.pxd

```diff
@@ -16,14 +16,40 @@ cdef extern from "<OpenMS/ANALYSIS/QUANTITATION/IsotopeLabelingMDVs.h>" namespac
 
         void isotopicCorrection(const Feature & normalized_feature, Feature & corrected_feature, 
           Matrix[double] & correction_matrix, const DerivatizationAgent & correction_matrix_agent) nogil except +
+            # wrap-doc:
+              #   This function performs an isotopic correction to account for unlabeled abundances coming from
+              #   the derivatization agent (e.g., tBDMS) using correction matrix method and is calculated as follows:
+              #   -----
+              #   :param  normalized_feature: Feature with normalized values for each component and unlabeled chemical formula for each component group
+              #   :param  correction_matrix:  Square matrix holding correction factors derived either experimentally or theoretically which describe how spectral peaks of naturally abundant 13C contribute to spectral peaks that overlap (or convolve) the spectral peaks of the corrected MDV of the derivatization agent
+              #   :param  correction_matrix_agent: name of the derivatization agent, the internally stored correction matrix if the name of the agent is supplied, only "TBDMS" is supported for now
+              #   :returns: corrected_feature: Feature with corrected values for each component
 
         void isotopicCorrections(
           const FeatureMap & normalized_featureMap, FeatureMap & corrected_featureMap,
           Matrix[double] & correction_matrix, const DerivatizationAgent & correction_matrix_agent) nogil except +
+            # wrap-doc:
```

Comment:
dots (File)

Reply:
The dots in the mulit line statement are still there - please have a look at the whole file.